decidim-core/spec/services/decidim/download_your_data_exporter_spec.rb

```diff
@@ -65,5 +66,70 @@ module Decidim
         end
       end
     end
+
+    describe "#readme" do
+      describe "the user" do
+        let(:help_definition_string) { "The username of the user" }
+
+        it_behaves_like "a download your data entity"
+      end
+
+      context "when the user has a user group" do
+        let!(:user_group) { create(:user_group, users: [user]) }
+        let(:help_definition_string) { "The username of the user" }
+
+        it_behaves_like "a download your data entity"
+      end
+
+      context "when the user has a follow" do
+        let!(:follow) { create(:follow, user:) }
+        let(:help_definition_string) { "The resource or space that is being followed" }
+
+        it_behaves_like "a download your data entity"
+      end
+
+      context "when the user has a notification" do
+        let!(:notification) { create(:notification, user:) }
+        let(:help_definition_string) { "The type of the resource that the notification is related to" }
+
+        it_behaves_like "a download your data entity"
+      end
+
+      context "when the user has a conversation" do
+        let!(:conversation) { create(:conversation, originator: user) }
+        let!(:message) { create(:message, conversation:) }
+        let(:help_definition_string) { "The messages of this conversation" }
+
+        it_behaves_like "a download your data entity"
+      end
+
+      context "when the user has a comment" do
+        let(:participatory_space) { create(:participatory_process, organization:) }
+        let(:component) { create(:component, participatory_space:) }
+        let(:commentable) { create(:dummy_resource, component:) }
+        let!(:comment) { create(:comment, commentable:, author: user) }
+        let(:help_definition_string) { "If this comment was a favour, against or neutral" }
+
+        it_behaves_like "a download your data entity"
+      end
+
+      context "when the user has a comment vote" do
+        let(:participatory_space) { create(:participatory_process, organization:) }
+        let(:component) { create(:component, participatory_space:) }
+        let(:commentable) { create(:dummy_resource, component:) }
+        let!(:comment) { create(:comment, commentable:) }
+        let!(:comment_vote) { create(:comment_vote, comment:, author: user) }
+        let(:help_definition_string) { "The weight of the vote (1 for upvote, -1 for downvote)" }
+
+        it_behaves_like "a download your data entity"
+      end
```

Comment:
I think this could be moved to comments engine, as it is related to comments ?

Reply:
Done at 692d2bb0f5597c4e06d6622020eb40d8f7d682ed